azure/sentinel/variables.tf

```diff
@@ -0,0 +1,148 @@
+###################
+# Global Variables
+###################
+
+variable "location" {
+  description = "The region in which resources will be created"
+  type        = string
+  default     = null
```

Comment:
Null region is invalid and will fail. Location is required or perhaps could be defaulted to "uksouth"?